.codeclimate.yml

```diff
@@ -2,3 +2,5 @@ exclude_paths:
 - vendor/**/*
 - ZeroBin/**/*
 - tests/**/* 
```

Comment:
It looks like there is no `tests` directory in the repo. Can you also remove this?